File: Modul-4/Soal-3/PRAK403-2410817320001-NazlaSalsabila.py
angka1, angka2 = map(int, input("Masukkan angka pertama dan kedua: ").split())

if angka1 < angka2:
    for i in range(angka1, angka2 + 1):
        print(i, angka2 - (i - angka1), end=" - " if i != angka2 else "\n")
else:
    for i in range(angka1, angka2 - 1, -1):
        print(i, angka2 + (angka1 - i), end=" - " if i != angka2 else "\n")

File: Modul-4/Soal-3/PRAK403-2410817320001-NazlaSalsabila.c
#include <stdio.h>

int main() {
    int angka1, angka2;
    printf("Masukkan angka pertama dan kedua: ");
    scanf("%d %d", &angka1, &angka2);

    if (angka1 < angka2) {
        for (int i = angka1; i <= angka2; i++) {
            printf("%d %d", i, angka2 - (i - angka1));
            if (i != angka2) {
                printf(" - ");
            }
        }
    }
    else {
        for (int i = angka1; i >= angka2; i--) {
            printf("%d %d", i, angka2 + (angka1 - i));
            if (i != angka2) {
                printf(" - ");
            }
        }
    }
    printf("\n");

    return 0;
}